phpBB/config/default/routing/routing.yml

```diff
@@ -24,5 +24,10 @@ phpbb_help_routing:
     resource: help.yml
     prefix: /help
 
+phpbb_mention_controller:
+    path: /mention
```

Comment:
Possible to rename the path like ``mention_json`` or ``mention_api``?

Reply:
I don't think there's a need in introducing a suffix with underscore.